src/ErrorPage.jsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useRef, useState } from "react";
+import { useRouteError, Link } from "react-router-dom";
+
+function ErrorPage() {
+  const occuredError = useRouteError();
+  const messageRef = useRef();
+  const [errorMessage, setErrorMessage] = useState("");
+  const [errorMessageReadingTime, setErrorMessageReadingTime] = useState(0);
+
+  useEffect(() => {
+    if (messageRef.current) {
+      const message = messageRef.current.innerText;
+      setErrorMessage(message);
+      const wpm = localStorage.getItem("wpm") || 202;
+      const readingTime = errorMessage.split(" ").length / (wpm / 60);
+      setErrorMessageReadingTime(Math.round(readingTime));
```

Comment:
```suggestion
      const message = messageRef.current.innerText;
      
      setErrorMessage(message);
      
      const wpm = localStorage.getItem("wpm") || 202;
      const readingTime = errorMessage.split(" ").length / (wpm / 60);
      
      setErrorMessageReadingTime(Math.round(readingTime));
```
이전에 논의한 개행 규칙에 맞게 작성 부탁드립니다!

Reply:
수정했습니다